sphinx/writers/latex.py

```diff
@@ -217,13 +219,6 @@ def __init__(self, language_code, use_polyglossia=False):
         self.supported = True
         super(ExtBabel, self).__init__(language_code or '')
 
-    def get_shorthandoff(self):
```

Comment:
Please follow our deprecation rule. I know this is minor and internal API. But sometimes extensions uses such an API. So let deprecate this before removing.

* Add deprecation warnings `warnings.warn()`
* Update CHANGES and deprecation list on `doc/extdev/index.rst`